CHANGELOG.rst

```diff
@@ -2,6 +2,13 @@
 CHANGELOG
 =========
 
+1.5.4
+=====
+* Add option to disable ganglia `extra_json = { "cfncluster" : { "ganglia_enabled" : "no" } }``
```

Comment:
extra `

Reply:
Fixing in https://github.com/awslabs/cfncluster/pull/525